docs: rephrase all sections (#191)
Open-Attestation/documentation-website: website/docs/developer-section/quickstart/create-verifiable-document-issuer.md

### Verifiable document components

Before starting on this code tutorial, it will be helpful to understand the components involved in the creation, issuance, and verification of a verifiable document.

You can read more about the components [here](/docs/integrator-section/verifiable-document/ethereum/document-store-overview).

## Overview

> You will build a single-page application, so that the user can perform the following:

Reviewer: ```suggestion
You will build a single-page application that performs the following
```

Reading through this page we changed a lot of things to "so that" which feels strange and repetitive, can we add some variations? not sure if the tense is in line with the rest

Author: updated to:

> You will build a single-page application with these features:
>
> 1. The users can connect to their MetaMask wallet.
> 2. They can deploy their own document store.
> ...

cd verifiable-document-issuer
```

> 2. Install the following packages, so that the application can interact with the blockchain.

Reviewer: ```suggestion
2. Install the following packages in order for the application to interact with the blockchain.
```

Author: updated to:

> 2. Install the following packages for the application to interact with the blockchain.

export default App;
```

> When you reload the app, the MetaMask extension should prompt you for your password and ask for permission, so that the site can access your accounts.

Reviewer: Can we revert this rephrase? In the context of permissions we should still say "allow"

Author: reverted

        
    * After successfully issuing the document, you can let the user download the `wrappedDocument` to be submitted for verification.
        
    * `Download` starts the download of the `wrappedDocument` and saves it on the user's devices.
        
    * `Verify` links to the verification site where the user can upload and verify the issued `wrappedDocument`.
        

>     * `Create Another` restarts the issuing process, so that the user can create and issue another `wrappedDocument`.

Reviewer: ```suggestion
    * `Create Another` restarts the issuing process to create and issue another `wrappedDocument`.
```

Author: updated
